experimental/ggml/benchmark_ggml.sh

```diff
@@ -0,0 +1,110 @@
+#!/bin/bash
+#
+# Copyright 2023 The OpenXLA Authors
+#
+# Licensed under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+#
+# Environment variables:
+# PYTHON: Python interpreter, default: /usr/bin/python3
+# OOBI_VENV_DIR: path to create Python virtualenv, default: ggml-benchmarks.venv
+# OOBI_TARGET_DEVICE: target benchmark device, can also be specified the first
+#   argument.
+# OOBI_BUILD_DIR: path to the GGMl build directory.
+# OOBI_OUTPUT: path to output benchmark results, can also be specified the
+#   second argument.
+#
+# Example usage:
+# ./benchmark_ggml.sh <target-device> <build-dir> <result-path>
+
+set -xeuo pipefail
+
+VENV_DIR="${OOBI_VENV_DIR:-ggml-benchmarks.venv}"
+PYTHON="${PYTHON:-/usr/bin/python3}"
```

Comment:
This might work in broader environments

```suggestion
PYTHON="${PYTHON:-"$(which python3)"}"
```

Reply:
Thanks, changed.